### ISSUE TITLE
Monero Community Workgroup Meeting: Saturday 27th May 2023 @ 15:00 UTC

### ISSUE THREAD

**Issue by plowsof** [monero-project/meta#838]
Location: [Libera.chat, #monero-community](https://libera.chat/) | [Matrix](https://matrix.to/#/#monero-community:monero.social?via=matrix.org&via=monero.social)

[Instructions for joining the monero.social Matrix server.](https://forum.monero.space/d/79-how-to-join-the-monero-core-team-matrix-server-web)

Time
15:00 UTC [Check your timezone](https://www.timeanddate.com/worldclock/converter.html)

Moderator: plowsof

Please reach out in advance of the meeting if you would like to propose an agenda item.

Proposed Meeting Items:

1. Introduction
2. Greetings
3. Community highlights    
    - [Jberman CCS update](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/359#note_21276) ~40% faster local wallet scanning using Koes [asynchronous design](https://gist.github.com/UkoeHB/4b3528a5c3a3134bd82d19a2bc6a8e87)
    - [Jeffro - transfer amount with fee included](https://github.com/monero-project/monero/pull/8861)
    - [Moneromooo CCS completed](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/192#note_21281)
    - [selstas reddit comment/summary](https://www.reddit.com/r/Monero/comments/134jbdt/comment/jigkpgj/?utm_source=share&utm_medium=web2x&context=3) of the recent remote node security advisory
    - [beta release for Monero subscriptions wallet](https://github.com/lukeprofits/Monero_Subscriptions_Wallet) - lukeprofits
    - [RingCT](https://rbrunner7.github.io/ring_ct.html) - Fictional Monero story by rbrunner
    - Channel with 2.14M subs shows Monero as an [alternative payment method](https://www.reddit.com/r/Monero/comments/13m7u2e/monero_mentioned_in_fireships_latest_video_huge/)
    - Haveno "a new release of the installers is out which now supports gold and silver. please give us a hand by helping us test the apps and network: https://github.com/haveno-dex/haveno/releases/tag/v0.0.6"
    - over the last 5 months [605.798 XMR worth of Bitcoin has been transformed into Monero]( https://nitter.it/UnstoppableSwap/status/1660260038718439430) - UnstoppableSwap.net
    - Monthly [Monero meetups](https://www.reddit.com/r/Monero/comments/13o0d9i/monero_t%C3%BCrkiye_meetup_25th_31st_may/) in  Türkiye - ajs
    - [Post-Mortem of 10-Block-Old Decoy Selection Bug 2023](https://github.com/monero-project/monero/issues/8872) - jeffro
    - [FeatherWallet update](https://old.reddit.com/r/FeatherWallet/comments/13r2urt/feather_wallet_249_released_includes_security/) - tobtoht
    - CryptoGrampy - [docker image](https://hub.docker.com/r/cryptogrampy/mymonero-web-js) for MyMonero.com LWS (hosting a public one for testing, DM for details)
    - Recent donations to bounties.monero.social [i2p baked into the GUI >10xmr](https://bounties.monero.social/posts/32/10-140m-i2p-baked-into-the-monero-gui) | [viewkey-based background refresh to reduce sync time >10xmr](https://bounties.monero.social/posts/41/10-503m-implement-viewkey-based-background-refresh-to-significantly-reduce-wallet-sync-time)
    - Mastering Monero [trending on hacker news](https://nitter.it/serhack_/status/1662120484228530177) SerHack
    - [How to start a censorship-resistant fundraiser with Monero](https://monerochan.news/article/13) - Spirobel interviews AnarkioCrypto
    - [CakeWallet update](https://nitter.it/cakewallet/status/1662200109625401344)
    - MySu.dev updated to v0.18.2.2 - to be confirmed
    - [Anonero](http://anonero5wmhraxqsvzq2ncgptq6gq45qoto6fnkfwughfl4gbt44swad.onion/#download) and Monerujo not updated - to be confirmed 
News: [Monero Observer](https://www.monero.observer/) - [Revuo Monero](https://revuo-xmr.com/) - [The Monero Standard](https://localmonero.co/the-monero-standard) - [Monero Moon](https://www.themoneromoon.com/)    
5. [CCS updates](https://ccs.getmonero.org/)    
  a. [VostoEmisio Animated Core Concept Videos](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/387)    
  b. [Wallet SDK for Android](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/388)    
  c. [Resolve abandoned outreach CCS](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/70#note_21324) - Ajs
6. Workgroup reports    
  a. Dev workgroup
  b. Localization workgroup
  c. Outreach workgroup
  d. Events workgroup - [MoneroKon 2023](https://github.com/MoneroKon/)
  e. Website workgroup
  f. Policy workgroup
  g. Research workgroup
  h. [Seraphis Migration workgroup](https://github.com/seraphis-migration)
7. Open ideas time    
8. Confirm next meeting date/time    

[Previous meeting including logs](https://github.com/monero-project/meta/issues/828)    

Meeting logs will be posted here afterwards.

**Comment by plowsof** [comment]
Logs 
```
10:09:32 <plowsof11> Meeting today in ~5 hours 15:00UTC https://github.com/monero-project/meta/issues/838

10:34:17 <ofrnxmr[m]> Matrix is very very slow

10:35:53 <ofrnxmr[m]> hopefully we can get her restarted before the meetings

10:40:12 <plowsof11> .ping is the bridge alive also

10:40:27 <plowsof> pong

10:59:28 <DanIsnotthemanBr> Might explain why only monero rooms are slow with the fluffychat

11:54:15 <plowsof11> thats alot of stickers from monerosupplies.com
https://www.reddit.com/r/Monero/comments/13sxmft/doing_my_part_bought_a_bunch_of_stickers_to_place/ , does
monerobull have an outreach update for us today regarding the monerochan plushies month?

13:18:25 <monerobull[m]> <plowsof11> "thats alot of stickers from..." <- A what

13:18:36 <monerobull[m]> Is there a meeting today

13:18:45 <monerobull[m]> I already have topia and Kon meeting ffs

13:19:31 <monerobull[m]> This monero stuff is more work than my actual job smh

13:24:50 <monerobull[m]> I do have a pretty cool announcement coming up though

13:26:15 <side-trips[m]> <monerobull[m]> "This monero stuff is more work..." <- I'll be mb stand-in rep if
needed. Consider it my attempt toward apprenticeship lol

13:26:58 <plowsof11> monerobull tried to unionise and has been replaced

13:27:15 <monerobull[m]> Huh

13:27:34 <monerobull[m]> No I'm actually trying to become Jeff bezos

13:27:44 <monerobull[m]> plowsof11: So the opposite

13:35:26 <DanIsnotthemanBr> Ceo of monero?

13:36:40 <monerobull[m]> That sounds like a liability but i have a mug that says #1 monero ceo

13:37:43 <monerobull[m]> Just checked, only says monero ceo

13:38:02 <monerobull[m]> #1 is probably vik

13:44:18 * monerobull[m] uploaded an image: (61KiB) <
https://libera.ems.host/_matrix/media/v3/download/matrix.org/KUZKEtgjNNbnLhbgiFrtVcnc/grafik.png >

14:49:39 <monerobull[m]> Ffs

14:50:10 <monerobull[m]> I just spent 20 minutes trying to figure out why i could reach
https://monerosupplies.com but not http

14:50:25 <monerobull[m]> It's because IVPN blocks http sites...

15:00:15 <plowsof11> Meeting time https://github.com/monero-project/meta/issues/838

15:00:28 <ofrnxmr[m]> Greetings

15:00:33 <plowsof11> hello

15:01:41 <Rucknium[m]> hi

15:01:43 <v1docq47[m]> hi

15:02:56 <ajs_[m]> Hi

15:03:04 <plowsof11> lets talk about recent events, firstly the vulnerability disclosure for 10 block old
decoys bug @ https://github.com/monero-project/monero/issues/8872 (which will soon be released on getmonero as
a blog post)

15:04:05 <monerobull[m]> Hello

15:04:15 <hinto[m]> hello

15:04:44 <plowsof11> several third party wallets have already updated: cakewallet:
https://nitter.it/cakewallet/status/1662200109625401344 , featherwallet:
https://old.reddit.com/r/FeatherWallet/comments/13r2urt and mysu.dev . it is strongly recommended everyone
update to the latest version of each - and cli/gui users to use v0.18.2.2 and above

15:05:06 <plowsof11> my nodes peer list has shown for the month of may 'more public rpc nodes using the latest
version than not' https://paste.debian.net/1281273/

15:05:30 <ofrnxmr[m]> Mysu.dev stackwallet.com also updated

15:05:41 <monerobull[m]> Mysu was the first 🥇

15:06:20 <Rucknium[m]> There is a suggestion by me in MRL to treat Monero's statistical protocols like its
cryptographic protocols are treated

15:06:22 <Rucknium[m]> Formal written specifications and code audits

15:06:25 <plowsof11> thanks for confirming, ofrnxmr had also contacted all of the above to inform them about
the issue.

15:06:26 <plowsof11> Monerujo will be the last?

15:06:51 <ofrnxmr[m]> monerujo and anonero pending

15:08:05 <Rucknium[m]> This is at least the third programming problem in the decoy selection algorithm (not
protocol/math/stats problem) in the last two years or so

15:08:56 <Rucknium[m]> If Monero's cryptography code had this many problems, it could be catastrophic.

15:09:39 <Rucknium[m]> Maybe I can convince jeffro to work on the C++ side and reserve-engineer a
specification for what the code actually does now.

15:10:02 <plowsof11> sounds like a priority , i wonder where it would fit into seraphis' plans (i know
jeffro256 also shares the sentiment of a full re-write of the algorithm)

15:10:27 <Rucknium[m]> We need to know what the current algorithm does anyway

15:10:35 <plowsof11> jeffro has recently finished his ccs ... perhaps something for the next one? 👀

15:10:43 <Rucknium[m]> Regardless of any plans for future rewrites

15:12:01 <ofrnxmr[m]> jeffro256:  ccs soon, i hope

15:12:48 <plowsof11> the Monero project gets flack for releasing disclosures on reddit/github issues - for
this one, we can pretend like there was a pre-release (to make sure it was factually correct, then we're going
to push it to a getmonero blog like professionals)

15:13:35 <Rucknium[m]> By the way, two of the problems were pretty classic programming mistakes (not at all a
statistics issue). This recent one was off-by-one error. Another one was integer division when it should have
been floating point division.

15:14:38 <plowsof11> vtnerd , and jberman also new ccs soon? (vtnerd has been looking into bulletproofs re-
write from haskell c++... [Jberman CCS update](https://repo.getmonero.org/monero-project/ccs-
proposals/-/merge_requests/359#note_21276) ~40% faster local wallet scanning using Koes [asynchronous
design](https://gist.github.com/UkoeHB/4b3528a5c3a3134bd82d19a2bc6a8e87)

15:15:14 <vtnerd> Yes, soon I will request one

15:16:19 <r4v3r23[m]> yo

15:16:37 <plowsof11> anything else to share? News: [Monero Observer](https://www.monero.observer/) - [Revuo
Monero](https://revuo-xmr.com/) - [The Monero Standard](https://localmonero.co/the-monero-standard) - [Monero
Moon](https://www.themoneromoon.com/)

15:16:37 <plowsof11> (after highlighting Ruckniums comments in bold)

15:17:13 <r4v3r23[m]> yes: https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/388

15:17:48 <plowsof11> we will get to that shortly yes

15:18:36 <plowsof11> moving on to the merge requests

15:18:46 <plowsof11>   a. [VostoEmisio Animated Core Concept Videos](https://repo.getmonero.org/monero-
project/ccs-proposals/-/merge_requests/387)

15:19:22 <plowsof11> they have created some short animated videos for monerokon (inside the Animated Videos
folder here https://cryptpad.disroot.org/drive/#/2/drive/view/e0aHmKq5gc5FqX-JXn7FgMQllZRdQfrqk+6ubQSjGBs/)

15:19:40 <monerobull[m]> Merge vosto

15:19:46 <monerobull[m]> Very good 👍

15:21:20 <plowsof11> they have made assurences that the produced video will have the same quality as this
example (not created by them but just a reference) https://www.yewtu.be/watch?v=pYOSfEgF32Y&ab_channel=XPLAI
and are putting an expiry date into their proposal of 4 months after funded to complete

15:21:59 <plowsof11> 9xmr to be raised through the CCS (not currently taking over the other animated videos
proposal)

15:22:21 <plowsof11> all things considered i like it too and will give a thumbs up

15:23:01 <hinto[m]> is the previous animation ccs still on-going?

15:23:16 <ofrnxmr[m]> hinto[m]: No

15:23:21 <hinto[m]> this seems like a good opportunity to re-use the funds

15:23:42 <ajs_[m]> vosto might also do post production video editing for monerokon 2023

15:23:46 <Rucknium[m]> VostoEmisio did two small paid graphic design projects for me. I was satisfied with the
result.

15:23:49 <ofrnxmr[m]> I believe the intent here is to try to do a better job at the same rate

15:24:03 <plowsof11> the reason for not re-using is they didnt specifically target taking over that ccs right
now

15:24:31 <plowsof11> ideally this would be a short test?

15:24:33 <ofrnxmr[m]> We can alwways choose to use the funds for it if we feel like it / after merging

15:24:43 <plowsof11> after the animated explainer ptsd

15:25:03 <plowsof11> providing they put the promises in this comment inside the proposal :
https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/387#note_21253

15:25:08 <ofrnxmr[m]> Iirc this is for 1 video to start, and will evolve into more if we accept

15:25:10 <monerobull[m]> plowsof11: That one was obvious trainwreck from the first demo though

15:25:18 <monerobull[m]> Vostos is actually looking awesome

15:26:11 <plowsof11> yes voting for a small ccs to evolve into 'possibly taking over the animated video ccs'
as ofrnxmr says above is my thoughts here

15:26:44 <plowsof11> is there anyone against it?

15:27:48 <ajs_[m]> +1

15:28:17 <plowsof11> the video will be (to be confirmed) about common misconceptions - first videos topic:
monero has an infinite supply , is this bad etc

15:28:46 <plowsof11> looks like its a merge on this one

15:28:46 <hinto[m]> no comment on this one

15:30:00 <plowsof11> moving on

15:30:01 <plowsof11>   b. [Wallet SDK for Android](https://repo.getmonero.org/monero-project/ccs-
proposals/-/merge_requests/388)

15:30:25 <cryptogrampy[m]> <plowsof11> "thanks for confirming, ofrnxmr..." <- Has anyone contacted mymonero?
Are they affected?

15:31:29 <cryptogrampy[m]> I just want to mention that I got mymonero-web-wallet working in a self-hosted way,
that works with self-hosted monero-lws server.  Spendss actually work as well ☺️

15:31:32 <Rucknium[m]> cryptogrampy: My guess is that MyMonero is not affected. I believe they still have
their own decoy selection code that is not a direct copy of wallet2's code. It would be good to check.

15:32:15 <plowsof11> also, giving your viewkey to them: they already know when you spend an output ?

15:32:23 <r4v3r23[m]> plowsof11: > <@plowsof:matrix.org>   b. [Wallet SDK for
Android](https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/388)

15:32:23 <r4v3r23[m]> >

15:32:23 <r4v3r23[m]> ive been testing this lib with valldrac to help identify bugs and get the demo app to a
stable state. android needs more monero support, since all the major wallets are already on or moving to
cross-platform frameworks. huge +1 from me

15:33:42 <plowsof11> thanks for testing r4v3r23

15:34:30 <ofrnxmr[m]> for me, sdk sccs largely depends on

15:34:30 <ofrnxmr[m]> 1. how this relates to molly ccs, if at all

15:34:30 <ofrnxmr[m]> 2. How this relates to existing java solutions such as monerujo

15:34:30 <ofrnxmr[m]> ...(truncated)

15:35:51 <r4v3r23[m]> ofrnxmr[m]: > <@ofrnxmr:monero.social> for me, sdk sccs largely depends on

15:35:51 <r4v3r23[m]> > 1. how this relates to molly ccs, if at all

15:35:51 <r4v3r23[m]> > 2. How this relates to existing java solutions such as monerujo

15:35:51 <r4v3r23[m]> ...(truncated)

15:36:15 <ofrnxmr[m]> Wth r u talking ab(ut

15:36:16 <ofrnxmr[m]> Depend on monerujo?

15:36:20 <r4v3r23[m]> you tell me

15:36:25 <ofrnxmr[m]> more options?

15:36:33 <r4v3r23[m]> "unless monerujo did the work already"

15:36:40 <r4v3r23[m]> whats wrong with more android options?

15:36:43 <ofrnxmr[m]> Read what i wrote and read the ccs

15:36:58 <ofrnxmr[m]> The proposer doesnt know if this is duplicate work

15:37:03 <plowsof11> kayabanerve spoke highly of this ccs in this comment here
https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/388#note_21318 (having spoke first
hand about issues including the security++ idea when validating blocks from remote nodes)

15:37:04 <r4v3r23[m]> fragementation? you prefer centralized options?

15:37:18 <r4v3r23[m]> ofrnxmr: your critique makes no sense

15:37:55 <r4v3r23[m]> a better, modern SDK for building android wallets, unrelated to molly. why be against?

15:38:04 <kayabanerve[m]> ofrnxmr: Molly has a distinct CSS for their integration, and as r4v3r23 noted, it's
not Molly specific.

15:38:12 <ofrnxmr[m]> only to you

15:38:18 <kayabanerve[m]> As for Monerujo, they definitively did not do the work present in the Molly efforts.

15:38:27 <r4v3r23[m]> this benefits the monero ecosystem as a hole

15:38:38 <r4v3r23[m]> s/hole/whole/

15:38:39 <plowsof11> monerujo dont exist until they update to v0.18.2.2, ticktock

15:38:53 <kayabanerve[m]> Molly still uses wallet2, yet barely. It's a complete re-architecture of the wallet
side.

15:38:57 <cryptogrampy[m]> Is monerujo still alive

15:39:08 <kayabanerve[m]> It's halfway between wallet2 and a complete rewrite (monero-serai).

15:39:11 <plowsof11> ofrnxmr himself updated monerujo apk to use an updated version of monero several months
ago

15:39:27 <r4v3r23[m]> so we only need 1 android option? i dont understand

15:39:32 <plowsof11> they slow AF

15:39:34 <ofrnxmr[m]> kayabanerve:  im referring to

15:39:34 <ofrnxmr[m]> > It is important to note that we have not conducted a comprehensive comparative
analysis yet between our SDK and existing Java libraries like Monerujo's xmrwallet or woodser's monero-java.

15:39:36 <r4v3r23[m]> are we pro centralization?

15:39:38 <kayabanerve[m]> That's how it's able to offer the amount of features, flexibility, and security it
declares.

15:39:46 <cryptogrampy[m]> Can valldrac up the ccs amount and add light wallet server support

15:40:22 <r4v3r23[m]> cryptogrampy[m]: hes open to more advanced features, first is to get a stable basic
version out

15:40:26 <kayabanerve[m]> ofrnxmr: Sorry if I'm missing context, I'm just joining in, but my point stands
AFAICT. Molly is a sweeping re-architecture to meet strict goals not achievable under existing works, which
simply bind wallet2.

15:40:50 <ofrnxmr[m]> the quote i posted is from the ccw

15:41:03 <r4v3r23[m]> any meaningful objections to a new Android wallet SDK?

15:41:16 <cryptogrampy[m]> If valldracs implementation makes it easier for people to build wallets than
monerujos existing code then I'm for it

15:41:20 <ofrnxmr[m]> Yes

15:41:21 <ofrnxmr[m]> > It is important to note that we have not conducted a comprehensive comparative
analysis yet between our SDK and existing Java libraries like Monerujo's xmrwallet or woodser's monero-java.

15:41:24 <kayabanerve[m]> Most of the features in the CCS aren't possible without messing with the guts of
wallet2. Accordingly, they're exclusive to Molly, since no one else has put in this level of effort yet (in
this context, Android).

15:41:32 <ofrnxmr[m]> valldrac themself claims they dont know (?)

15:42:34 <r4v3r23[m]>  ofrnxmr: dont donate if you have issues. i fail to see reasons to block this CCS

15:42:36 <plowsof11> note that this is the first time the idea has been discussed - and iirc valldrac also
wants to update the demo?

15:43:00 <r4v3r23[m]> plowsof11: he his, and ive been testing the apk

15:43:13 <kayabanerve[m]> @woodser's work on monero-java would be the best-case existing work, due to their
work on APIs, yet it's monero-java, not monero-android.

15:43:20 <kayabanerve[m]> Hm. I'll invite them to the conversation.

15:43:33 <ofrnxmr[m]> Thank you

15:44:38 <woodser[m]> hello there 👋

15:44:47 <kayabanerve[m]> I'm unsure if they immediately have the time, yet they can comment now how monero-
java is comparable to the proposed Molly work.

15:45:04 <kayabanerve[m]> Yo. Able to see the prior messages or need someone to bring down context?

15:45:37 <woodser[m]> yeah I can see prior messages

15:45:50 <woodser[m]> distilling the context is also nice :)

15:45:54 <kayabanerve[m]> Also, I could be wrong, yet AFAIK Monerujo doesn't have a library. It's an app which
integrates wallet2.

15:45:55 <r4v3r23[m]> note this isnt just an monero lib, its and SDK with a working demo app that devs can
base their wallets on

15:46:06 <kayabanerve[m]> I'm not trying to say it can't be used as a lib. I'm noting Monerujo isn't trying to
be a lib AFAIK.

15:46:34 <r4v3r23[m]> its makes creatong hugh quality android wallets significantly easier

15:46:34 <kayabanerve[m]> Molly CCS for an effective Monero Android SDK with a variety of security and
functionality improvements over wallet2.

15:46:35 <r4v3r23[m]> s/creatong/creating/, s/hugh/high/

15:46:40 <kayabanerve[m]> https://repo.getmonero.org/monero-project/ccs-proposals/-/merge_requests/388

15:46:57 <kayabanerve[m]> ofrnxmr: noted how valldrac themselves noted a lack of comparisons.

15:47:07 <kayabanerve[m]> * of comparisons between existing work and the proposed.

15:47:39 <r4v3r23[m]> devs still need to create JNI bindings when using monerujos lib

15:48:05 <plowsof11> i like that maintenance is added into the ccs and valdrac is going to put an expiry on it
so if its not completed in a reasonable amount of time, funds released. A comment from woodser would be great.
as we have 13 minutes left and another idea to discuss, seems logical to wait for the comparison and discuss
again later?

15:48:09 <kayabanerve[m]> I commented how existing work (Monerujo and monero-java) are wallet2 bindings.
Molly's proposal significantly re-impls wallet2 functionality to achieve its goals. I also ack'd xmr-java's
API work, and then brought you in to further compare the two.

15:48:24 <kayabanerve[m]> That's the summary, woodser ;)

15:48:30 <r4v3r23[m]> SDK is an upgrade in terms of development

15:50:22 <plowsof11> we can move on to the next idea in the meantime with 10 minutes left

15:50:35 <plowsof11>   c. [Resolve abandoned outreach CCS](https://repo.getmonero.org/monero-project/ccs-
proposals/-/merge_requests/70#note_21324) ajs_

15:51:32 <ajs_[m]> hi

15:52:11 <r4v3r23[m]> plowsof: re: 388 looks like majority in favor with no real objections

15:52:23 <ajs_[m]> the idea is to set up Mobilizon instance and promote meet ups

15:52:36 <plowsof11> nioc and midipoet had a question or 2 in those comments

15:53:05 <ofrnxmr[m]> How much of 36 xmr goes towards the first trip

15:53:09 <ofrnxmr[m]> The monerokon trip

15:53:45 <ajs_[m]> i was going to use some of the funding for a trip to prague and attend events durnig PBW,
but i am not sure I'll stay the whole week

15:54:06 <ajs_[m]> i might use monerokon funds for a quick trip for a few days

15:54:24 <ofrnxmr[m]> Including printing of flyers, stickers, and t-shirts to giveaway at meetup

15:54:57 <ajs_[m]> i think we can put off a decision

15:55:11 <ajs_[m]> and flush out the ideas more

15:55:35 <ajs_[m]> thats all

15:55:56 <midipoet> i just wanted to make sure the proposed meetups were advertised on meetup.com or whatever
the local/regular meetup website it (even if facebook), as we want to use those meetups to get new people in.
That is all really. Other than that i support the funds being used (though 36 XMR is quite a bit of fiat).

15:56:31 <plowsof11> guestimate of the funds required for each -  if i remember the trip will cost 3kusd? has
support from nioc,midipoet,geonic and maybe scott already judging by those comments. ajs is also: planning
monthly [Monero
meetups](https://www.reddit.com/r/Monero/comments/13o0d9i/monero_t%C3%BCrkiye_meetup_25th_31st_may/) in
Türkiye - ajs

15:57:10 <ofrnxmr[m]> So around 20/36 xmr - does this inckude the giveaways?

15:57:18 <ajs_[m]> 3k for the full week

15:57:26 <ajs_[m]> but i will make a short trip instead

15:57:44 <ajs_[m]> and bill monerokon funds

16:00:10 <plowsof11> so we need to know where the 36 xmr plays into this all (it you could itemise each /
flesh it out?) is that the plan moving forward?

16:00:37 <ajs_[m]> put the idea on hold

16:01:01 <ajs_[m]> when i have more time I'll put together a more detailed proposal

16:01:16 <plowsof11> ok thank you

16:02:57 <plowsof11> to summarise, VostoEmisio Animated Core Concept Videos -> merge. Android Wallet sdk ->
more support than not, i would feel comfortable waiting on woodsers feedback also (its also the first time
weve discussed it). Ajs taking over outreach -> on hold.


```
Automated by [this](https://github.com/plowsof/post-libera-meeting-logs)

[continued](https://libera.monerologs.net/monero-community/20230527#c249577)
